packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -16,6 +17,27 @@ interface State {
   data: DataFrame[];
 }
 
+const getStyles = () => {
+  return {
+    gfTableInputCsv: css`
+      position: relative;
+      textarea {
+        height: 100%;
+        width: 100%;
+      }
+      footer {
+        position: absolute;
+        bottom: 15px;
+        right: 15px;
+        border: 1px solid #222;
+        background: $online;
```

Comment:
This line won't work, as there is not $online variable. Please use theme colors.